node-backend/.env

```diff
@@ -0,0 +1,15 @@
+HTTP_PORT= 
```

Comment:
it's a good practice to have `.env.sample` file with sensible defaults against environment variables.